homeassistant/components/xiaomi_miio/vacuum.py

```diff
@@ -377,3 +393,21 @@ def update(self):
             _LOGGER.error("Got OSError while fetching the state: %s", exc)
         except DeviceException as exc:
             _LOGGER.warning("Got exception while fetching the state: %s", exc)
+
+    async def async_clean_zone(self,
+                               zone,
+                               repeats=1):
+        """Clean selected area for the number of repeats indicated."""
+        _LOGGER.debug("Zone to clean: %s repeats: %s", zone, repeats)
+        from miio import DeviceException
+        _LOGGER.debug("Original zone: %s", zone)
```

Comment:
Is this necssary? You are printing `zone` thrice here, first above in "zone to clean", then here and then again below.

Reply:
fixed. 
two log was removed